scripts/loader

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh -x
+set +m
```

Comment:
Add a comment to this.

"Disable job control so that all child processes run in the same process group as the parent"